whirlpool/oven.py

```diff
@@ -275,7 +274,9 @@ def get_oven_cavity_exists(self, cavity: Cavity):
 
     # todo: persist the kitchen timer objects in the object
     def get_kitchen_timer(self, timer_id=1):
-        timer = KitchenTimer(appliance=self, timer_id=timer_id)
+        timer = KitchenTimer(
+            appliance=self, app_manager=self._app_manager, timer_id=timer_id
+        )
```

Comment:
```suggestion
        timer = KitchenTimer(appliance=self, timer_id=timer_id)
```

Reply:
Fixed in 046dd236a6a5aec3a2a8bad9062df3d92bc41a0b